CotEditor/zh-Hans.lproj/DocumentInspectorView.strings

```diff
@@ -27,7 +27,7 @@
  */
 
 /* Class = "NSTextFieldCell"; title = "Dofument File"; ObjectID = "5tP-OA-Ujw"; */
```

Comment:
I guess it should be "Document" instead of "Dofument". But I just leave it there without modification.